src/components/AccountForms/RegistrationForm/ChipComponent.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import useStyles from './RegistrationForm.styles';
+import './registration.css';
+import { InputLabel, Chip } from '@material-ui/core';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+
+const ChipComponent = ({ register, unregister, watch, setValue, errors }) => {
+  const classes = useStyles();
+  const watchRole = watch('role_id', '');
+  const handleClick =
+    ('onClick',
+    (event) => {
+      const id = event.currentTarget.id;
+      setValue('role_id', id, {
+        shouldDirty: true,
+        shouldValidate: true,
+      });
+    });
```

Comment:
let's refactor this code block:

`const handleClick = val =>
    setValue('role_id', val, { shouldDirty: true, shouldValidate: true });`